app.js

```diff
@@ -0,0 +1,84 @@
+const React = require("react");
+const ReactMarkdown = require("react-markdown");
```

Comment:
Don't think we've discussed as a team what our default `prettier` settings should be – the EB version (i.e. single quotes, trailing commas) or the tool defaults? Either way we should maybe add a `.pretterrc` to this repo

Reply:
Oh sorry, I totally meant to add one - I'll do that